src/frontends/onnx/frontend/src/frontend.cpp

```diff
@@ -89,11 +90,14 @@ std::shared_ptr<ngraph::Function> FrontEnd::convert(const InputModel::Ptr& model
         return function;
     }
 
-    return model_onnx->convert();
+    const auto& converted_model = model_onnx->convert();
+    normalize(converted_model);
```

Comment:
FYI: In the expected flow of Frontends, `normalize` should be the last part of `convert_partially`. `convert_partially` should be called from `convert`. The only diff between `convert_partially` and `convert` is checking if we haven't any unsupported ops in the result model. Until we haven't `convert_partially` in ONN FE (also planned for 23.1) it's OK.

Reply:
As I understand, `normalize` should not be called inside `convert_partially`, only inside `convert`. `convert` is basically equal to calling `convert_partially` + `normalize` and verifying that no unconverted ops left.